doc/compiling.tex

```diff
@@ -104,27 +98,29 @@ \subsection{Compilation flags}
 \subsection{Building on Debian or Ubuntu Linux}
 \label{sec:compiling-ubuntu}
 You'll first need to install a number of dependencies. It is assumed here that you are using a
-recent version of Ubuntu Linux (Precise Pangolin / 12.04 LTS or later), hence some of the
+recent version of Ubuntu Linux (Xenial Xerus / 16.04 LTS or later), hence some of the
 package may be named differently if you are using Debian Linux or another Ubuntu version.
 
 First, run
 \begin{shell}
-$\text{\$}$ sudo apt-get install build-essential scons mercurial qt4-dev-tools libpng12-dev
+$\text{\$}$ sudo apt-get install build-essential scons git libpng12-dev
```

Comment:
qt5-dev-tools?

Reply:
It's into a different paragraph now, see below (line 114). I also changed it to `qt5-default` because it is really easy to run into issues if one has Qt4 installed (it sets `QT_SELECT` default preference to `Qt5`). Anyway, I added a footnote explaining how to work around it, just to be sure.